### PR TITLE
Fixed the version of operator-custom-metrics

### PR DESCRIPTION
**Pull request by arjunrn**
Updated to latest version of `operator-custom-metrics`. Also fixed the value of the namespace passed to metrics configuration.

/assign @jharrington22 @2uasimojo

### CODE REVIEW

**Comment by 2uasimojo** [comment]
Didn't 0.4.1 fix the problem? https://github.com/openshift/operator-custom-metrics/tags

**Comment by arjunrn** [comment]
@2uasimojo I've updated the library version.

File: config/config.go [hunk 1]
```diff
@@ -16,5 +16,5 @@ package config
 
 const (
 	OperatorName = "osd-metrics-exporter"
-	OperatorNamespace = "osd-metrics-exporter"
+	OperatorNamespace = "openshift-osd-metrics"
```
Comment (2uasimojo) [inline config/config.go:19]: Is this an unrelated change?

**Comment by fahlmant** [comment]
@arjunrn Is 0.4.1 broken?

**Comment by arjunrn** [comment]
> @arjunrn Is 0.4.1 broken?

@fahlmant No, I thought 0.4.0 was the last released version. I didn't see you had released 0.4.1.

File: config/config.go (hunk 1 above)
Comment (arjunrn) [reply config/config.go:19]: Yeah. But it was broken and it's related to the cleanup. I've mentioned it in the PR description.

**Comment by fahlmant** [comment]
ah np :+1: Just wanted to verify
